garak/resources/gcg/attack_manager.py

```diff
@@ -1932,7 +1923,5 @@ def get_goals_and_targets(
 
     assert len(goals) == len(targets)
     assert len(test_goals) == len(test_targets)
-    logger.debug("Loaded {} train goals".format(len(goals)))
-    logger.debug("Loaded {} test goals".format(len(test_goals)))
```

Comment:
btw how is the goal string set? a docstring or `docs/` entry somewhere should cover this (maybe in one terse sentence)

Reply:
I ought to write a full docs entry for each of these. Will take that as an action item and open another PR.